scripts/screens/organizational_screens.py

```diff
@@ -189,6 +203,19 @@ def screen_switches(self):
         self.open_data_directory_button.hide()
         self.closebtn.hide()
 
+        self.update_button = UIImageButton(scale(pygame.Rect((1154, 50), (382.5, 75))), "",
+                                             object_id="#update_button", manager=MANAGER)
+        self.update_button.visible = 0
+
+        try:
+            global has_checked_for_update
+            if game.settings['check_for_updates'] and not has_checked_for_update and has_update(UpdateChannel(get_version_info().release_channel)):
```

Comment:
There is still an issue with the enum I assume. (updated all requirements in requiremnts.txt) I would suggest to make the StrEnum as a normal Enum and just write the strings yourself.
![image](https://user-images.githubusercontent.com/110975120/230790920-d533e458-b5ec-4c32-8d69-b0a6b196e769.png)